src/briefcase/integrations/android_sdk.py

```diff
@@ -1281,3 +1270,15 @@ def logcat(self):
             # Exit normally since the user was instructed to use CTRL+C.
             if e.returncode not in {-2, 0xC000013A}:
                 raise BriefcaseCommandError("Error starting ADB logcat.") from e
+
+    def pidof(self, package):
+        """Return the PID of the given app as a string, or None if it isn't
+        running."""
```

Comment:
The PID is returned as a string because it'll only ever be used as a string.

Reply:
But what if I want to perform complex arithmetic analysis on my PID??!?